scale/.gitignore

```diff
@@ -0,0 +1,91 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This is a good .gitignore! Let's just move it to the root of the repo so it applies to the whole repo?